source

```diff
@@ -100263,6 +100272,15 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
      </div>
 
     <tr>
+      <td><dfn data-x="parse-error-missing-doctype">missing-doctype</dfn>
+      <td>
+        <p>This error occurs if the parser encounters any content other than <span
+        data-x="syntax-comments">comments</span> or <span>ASCII whitespace</span> before <span
+        data-x="syntax-doctype">DOCTYPE</span> in the beginning of the document, i.e. DOCTYPE is not
```

Comment:
data-x="syntax-doctype">DOCTYPE</span> ~in~ **at** the beginning of the document, i.e. DOCTYPE is not